cmake/generic.cmake

```diff
@@ -349,10 +349,17 @@ function(cc_test TARGET_NAME)
     set(oneValueArgs "")
     set(multiValueArgs SRCS DEPS ARGS)
     cmake_parse_arguments(cc_test "${options}" "${oneValueArgs}" "${multiValueArgs}" ${ARGN})
+    if(WIN32)
+      list(APPEND win32_deps shlwapi)
```

Comment:
in face, shlwapi should be add to gtest, it's a gtest dependency instead of generic.cmake

Reply:
Yes and this seems a easiest way to make the minimal change to the source code.